### DISCUSSION TITLE
Since neorg has it's own syntax, can we swap quote and marker?

### DISCUSSION THREAD

**Discussion by txtyash** [nvim-neorg/neorg#256] [General]
I think it'd be nice to have `|` for quote which uses `❚` unicode symbol in
muted colors obviously.

And marker could be used with `>` and fine with any glyph.

**Comment by mrossinek** [comment]
Thanks, @zim0369, for the suggestion! We had a quick discussion over on our discord, but we feel like that this is not a change we want to implement. Yes, it is true, that norg is its own format and we're not afraid to do things differently, but we still want to be wary of our roots and `>` is being used for quotes _everywhere_. That is not only markdown but even basic email, 4chan, etc use this.

So preserving some history seems like the better way to go in this scenario.
